tests/sentry/utils/audit/tests.py

```diff
@@ -37,3 +43,62 @@ def test_audit_entry_frontend(self):
         assert entry.actor == req.user
         assert entry.actor_key is None
         assert entry.ip_address == req.META['REMOTE_ADDR']
+
+
+class DeletedEntryTest(APITestCase):
+
+    def test_deleted_organization(self):
+        user = self.create_user()
+        organization = self.create_organization(slug='slug123456789', owner=user)
+
+        path = '/organizations/%s/remove/' % (organization.slug)
```

Comment:
I think we prefer the [`format`](https://docs.python.org/2/library/stdtypes.html#str.format) string method these days.